Sort3i4/.vs/ProjectSettings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "CurrentProjectSetting": "Brak konfiguracji"
```

Comment:
dodaj proszę gitignora lokalnie i wywal ze śledzenia wszystkie pliki które nie są kodem, który mam oglądać. 42 pliki? co to jest.